internal/querynode/collection.go

```diff
@@ -170,15 +185,21 @@ OUTER:
 
 // getPChannels get physical channels of collection
 func (c *Collection) getPDeltaChannels() []Channel {
+	c.channelMu.RLock()
+	defer c.channelMu.RUnlock()
 	return c.pDeltaChannels
```

Comment:
shoud do copy before return to avoid race

Reply:
fixed